bool color
int8 ply
---
string move
